app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,58 @@
+<h1 class='title'> Recipe 2</h1>
+<div class='recipe-detail-container'>
+ <div class='preparation'>
+   <p class='preparation-info'>Preparation time: <%= @recipe.preparation_time.strftime('%H:%M') %> hours </p>
+   <p class='preparation-info'>Cooking time: <%= @recipe.cooking_time.strftime('%H:%M') %>hours</p>
+   <p class='preparation-info'>Steps go here...</p>
+   <button class='add-btn'>Generate shopping list</button> 
+ </div>
+ <div class='public-info'>
+ <div class='public-container'>
+ <div >
+  <p class='public-title'>
+  Public
+  </p>
+  </div>
+  <div class='public-form'>
+   <%= form_with(model: @recipe, url: toggle_public_recipe_path(@recipe), method: :patch, local: true) do |form| %>
+    <button type="submit" class="icon-button">
+      <% if @recipe.public? %>
+        <%= icon('fas', 'toggle-on', class: 'fa-2x') %>
+      <% else %>
+        <%= icon('fas', 'toggle-off', class: 'fa-2x') %>
+      <% end %>
+    </button>
+  <% end %>
+  </div>
+  
+  </div>
+
+  
+  <% if can?(:create, @recipe) %>
+  <%= link_to new_food_path do %>
+    <button class='add-btn'>Add food</button>
+  <% end %>
+ <% end %>  
+ </div>
+</div>
+
+<table>
+ <tr>
+ <th>Food</th>
+ <th>Measurement Unit</th>
+ <th>Value</th>
+ <th>Actions</th>
+ </tr>
+ <% @recipe_foods.each do |recipe_food| %>
+ <tr>
+ <td><%= recipe_food.food.name if recipe_food.food %></td>
+ <td><%= recipe_food.food.quantity if recipe_food.food %> </td>
+ <td><%= recipe_food.food.price if recipe_food.food %></td>
+ <td>
+  <button>Modify</button>
+  <button>Remove</button>
+ </td>
+ </tr>
+ <% end %>
+ <tr>
+</table>
```

Comment:
- [ ] [MANDATORY] Just as mentioned above, in the recipe show page will not show a thing since we were not able to see the recipes page and get the button for creating a recipe.